kolibri/core/assets/src/views/PaginatedListContainer.vue

```diff
@@ -74,65 +66,108 @@
-      },
-      startRange() {
-        return (this.currentPage - 1) * this.itemsPerPage;
+        return this.totalPageNumbers;
       },
       visibleStartRange() {
-        return Math.min(this.startRange + 1, this.numFilteredItems);
+        return this.currentPageNumber;
       },
       endRange() {
         return this.currentPage * this.itemsPerPage;
       },
       visibleEndRange() {
         return Math.min(this.endRange, this.numFilteredItems);
       },
-      visibleFilteredItems() {
-        return this.filteredItems.slice(this.startRange, this.endRange);
-      },
       previousButtonDisabled() {
-        return this.currentPage === 1 || this.numFilteredItems === 0;
+        return this.currentPageNumber === 1 || this.numFilteredItems === 0;
       },
       nextButtonDisabled() {
         return (
-          this.totalPages === 1 ||
-          this.currentPage === this.totalPages ||
+          this.totalPageNumbers === 1 ||
+          this.currentPageNumber === this.totalPageNumbers ||
           this.numFilteredItems === 0
         );
       },
     },
     watch: {
-      visibleFilteredItems: {
-        handler(newVal) {
-          this.$emit('pageChanged', {
-            page: this.currentPage,
-            items: newVal,
-          });
+      filterInput: {
+        handler() {
+          this.currentPageNumber = 1;
+          this.get_users();
+        },
+      },
+      roleFilter: {
+        handler() {
+          this.currentPageNumber = 1;
+          this.get_users();
         },
-        immediate: true,
       },
     },
     methods: {
+      get_users() {
+        const facilityId = store.getters.activeFacilityId;
+        FacilityUserResource.fetchCollection({
+          getParams: {
+            member_of: facilityId,
+            page_size: this.itemsPerPage,
+            page: this.currentPageNumber,
+            search: this.filterInput,
+            exclude_member_of: !this.excludeMemberOf ? '' : this.excludeMemberOf,
+            user_type:
+              !this.roleFilter || this.roleFilter.value === 'all' ? '' : this.roleFilter.value,
+            exclude_user_type: this.userAssignmentType === 'coaches' ? 'learner' : '',
+          },
+          force: true,
+        }).then(
+          users => {
+            this.currentPageNumber = users.page;
+            this.userList = users.results;
+            this.totalPageNumbers = users.total_pages;
+            this.totalItems = users.count;
+          },
+          error => {
+            store.dispatch('handleApiError', error);
+          }
+        );
+      },
       changePage(change) {
-        // Clamp the newPage number between the bounds if browser doesn't correctly
-        // disable buttons (see #6454 issue with old versions of MS Edge)
-        this.currentPage = clamp(this.currentPage + change, 1, this.totalPages);
+        this.currentPageNumber += change;
+        this.get_users();
```

Comment:
why did you remove the clamping that fixes the IE and edge issue? can't happen anymore?